galaxyxml/tool/import_xml.py

```diff
@@ -25,12 +25,10 @@ def _init_tool(self, xml_root):
         for child in xml_root:
             if child.tag == "description":
                 description = child.text
-            elif child.tag == "command":
-                executable = child.text.split()[0]
-                command = child.text
```

Comment:
it took me a while to understand this, it's because now the command is parsed in `Command`, rather than expecting the entire thing to be set from the start. I was also unhappy it was re-ordered since it made the diff unnecessarily confusing.

Reply:
not a deliberate or useful reorder - now less reordered.